src/System/i18n/locales/en-US/translation.json

```diff
@@ -103,11 +103,15 @@
     "markAllAsRead": "Mark all as read"
   },
   "artQuizPage": {
+    "title": "Art Taste Quiz",
     "welcomeScreen": {
-      "title": "Art Curator Quiz",
       "subtitle": "See more of what you love. Rate artworks to discover your taste profile and get recommendations tailored to you.",
       "getStartedButton": "Start the Quiz",
       "skipButton": "Skip"
+    },
+    "loadingScreen": {
+      "calculatingResults": "Calculating Results...",
+      "resultsComplete": "Results Complete"
```

Comment:
**praise:** following best practices

You love to see additions to Force get i18n love right from the start - bravo!! 🙌 

*format via [Conventional Comments](https://conventionalcomments.org/)*